CHANGELOG.md

```diff
@@ -0,0 +1,50 @@
+<a name="v1.1.0"></a>
+## v1.1.0
+
+### Build
+- **deps:** bump [@fortawesome](https://github.com/fortawesome)/fontawesome-svg-core
+- **deps:** bump [@fortawesome](https://github.com/fortawesome)/react-fontawesome from 0.1.11 to 0.1.13
+- **deps:** update jest to 26.6.3
+- **deps-dev:** bump [@types](https://github.com/types)/enzyme from 3.10.5 to 3.10.8
+- **deps-dev:** bump webpack-dev-server from 3.11.0 to 3.11.2
+
+### Ci
+- add types for conventional commits
+- update to include coverage
+- sync dependencies
+- add simple build for ui-toolkit
+
+### Docs
+- add status badges
+- add release disclaimer
+
+### Feat
+- add cors support
+- hide MEBx password related fields for client control mode in profile control
+
+### Fix
+- upgrade ws from 7.2.0 to 7.3.1
+- **ciraconfig:** root cert now uses mps server address
+- **ciraconfig:** mps root cert autoload
+- **ciraconfig:** mps root cert autoload
+- **ciraconfigform:** changed mpsServer prop mandatory
+- **ciraconfigform:** resloved review comments
```

Comment:
correct spelling?

Reply:
I'll fix it...this time...but this is autogenerated from the commit messages, so next time I generate it will be there again :)